demo/site/src/util/graphQLClientMiddleware.ts

```diff
@@ -0,0 +1,15 @@
+import { createFetchWithDefaults, createGraphQLFetch } from "@comet/cms-site";
+
+export function createGraphQLFetchMiddleware() {
+    return createGraphQLFetch(
```

Comment:
nit: I'd still call it `createGraphQLFetchLibrary` as a distinction to the project's `createGraphQLFetch`. At first glance, I thought `createGraphQLFetchMiddleware` uses the other `createGraphQLFetch` which is confusing

```suggestion
import { createFetchWithDefaults, createGraphQLFetch as createGraphQLFetchLibrary } from "@comet/cms-site";

export function createGraphQLFetchMiddleware() {
    return createGraphQLFetchLibrary(
```